doc.cfm

```diff
@@ -31,7 +31,12 @@
 			<cfset ArrayAppend(possible, i)>
 		</cfif>
 	</cfloop>
-	<cfset data = {name = url.name, description="Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a pull request.", type="404", related=possible}>
+	<cfset data = {
+		name = url.name,
+		description = "Sorry we don't have any docs matching that name. If we should have a doc for this, please send us a <a href=""https://github.com/foundeo/cfdocs/edit/master/data/en/" & url.name & ".json"">pull request</a> on github.\n\nYou can easily access functions and tags using an url like <a href=""http://cfdocs.org/hash"">cfdocs.org/hash</a>. Just hit <code>/tag-name</code> or <code>/function-name</code> or use the search box above.\n\nMaybe it's just a spelling mistake. Please make sure you did not misspell it before you open a pull request or issue!",
```

Comment:
The link will always resolve to a 404: https://github.com/foundeo/cfdocs/edit/master/data/en/foo.json